cli/teos_cli.py

```diff
@@ -7,10 +7,11 @@
 from uuid import uuid4
 from coincurve import PublicKey
 from getopt import getopt, GetoptError
+from http.client import HTTPException
```

Comment:
No need. Just check for status code != 200 and report. There are some examples in `process_post_response`.